src/app/myworkflows/sidebar-accordion/github-apps-logs/github-apps-logs.component.ts

```diff
@@ -206,4 +222,22 @@ export class GithubAppsLogsComponent extends Base implements OnInit, AfterViewIn
       this.showContent = 'table';
     }
   }
+
+  mapSortOrder(rawSortOrder: string): 'asc' | 'desc' {
```

Comment:
I'm not entirely sure why this is needed 
Lifted from tool paging which must have had the same issue 
https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/tool-lister.ts#L112-L125